packages/vaadin-lumo-styles/src/components/multi-select-combo-box-chip.css

```diff
@@ -0,0 +1,87 @@
+:host {
+  font-size: var(--lumo-font-size-xxs);
+  line-height: 1;
+  color: var(--lumo-body-text-color);
+  border-radius: var(--lumo-border-radius-s);
+  background-color: var(--lumo-contrast-20pct);
+  cursor: var(--lumo-clickable-cursor);
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+:host([disabled]) {
+  background-color: var(--lumo-contrast-10pct);
+}
+
+:host([focused]) [part='remove-button'] {
+  color: inherit;
+}
+
+:host([slot='overflow']) {
+  position: relative;
+  min-width: var(--lumo-size-xxs);
+  margin-inline-start: var(--lumo-space-s);
+}
+
+:host([slot='overflow'])::before,
+:host([slot='overflow'])::after {
+  position: absolute;
+  content: '';
+  width: 100%;
+  height: 100%;
+  border-left: calc(var(--lumo-space-s) / 4) solid var(--lumo-contrast-30pct);
```

Comment:
I had to switch to the shorthand because Chrome couldn't parse the original version for some reason:

```css
border-left: calc(var(--lumo-space-s) / 4) solid;
border-color: var(--lumo-contrast-30pct);
```